doc/source/multivisit.rst

```diff
@@ -3,4 +3,535 @@
+
+    skycell-p1889x07y19 25
+    skycell-p1889x07y20 8
+    skycell-p1970x15y03 8
+    skycell-p1970x15y02 17
+    skycell-p1970x16y02 8
+
+All subsequent examples will use the exposures for SkyCell **skycell-p1889x07y19**.  The member exposures filenames
+can be written out to its own input file named **skycell-p1889x07y19_mvm_input.txt** for MVM processing using:
+
+.. code:: python
+
+    with open('skycell-p1889x07y19_input.txt', 'w') as fout:
+        _ = [fout.write(f'{file}\n') for file in p1889_scells['skycell-p1889x07y19'].members]
+
+    mpf.generate_poller_file('skycell-p1889x07y19_input.txt', poller_file_type='mvm', output_poller_filename='skycell-p1889x07y19_mvm_input.txt', skycell_name='skycell-p1889x07y19')
+
+
+
+Defining SkyCell Layers
+------------------------
+Defining the SkyCell for a region on the sky allows for the identification of all exposures that overlap that WCS.
+However, creating a single mosaic from data taken with different detectors and filters would not result in a
+meaningful result.  Therefore, the exposures that overlap each SkyCell get grouped based on the detector and filter used
+to take the exposure to define a 'layer' of the SkyCell.  Each layer can then be generated as the primary basic image
+product for each SkyCell.  Exposures taken with spectroscopic elements, like grisms and prisms, and exposures taken of
+moving targets can not be used to create layers due to the inability to align them with the rest of the observations.
+Therefore, only images taken with standard filters (like the WFC3/UVIS F275W filter) will be used to define SkyCell
+mosaics (layers).
+
+The default plate scale for all MVM image products for each SkyCell has been defined as 0.04"/pixel to match the higher
+resolution imaging performed by the WFC3/UVIS detector.  However, WFC3/IR data suffers from serious resampling artifacts
+when drizzling IR data to that plate scale. So in addition to creating IR mosaics at the 0.04"/pixel 'fine' plate scale,
+IR mosaics are also generated at a 'coarse' plate scale of 0.12"/pixel to minimize the resampling artifacts while also being easily
+scaled to the 'fine' plate scale mosaics.
+
+SkyCell Layers Example
+'''''''''''''''''''''''
+For example, observations were taken with Proposals 12286 and 12903 using both the ACS and WFC3 cameras and
+multiple filters. The ACS observations were taken
+with the ACS/WFC detector using the F775W and F850LP filters, while the WFC3 observations were taken using the IR detector
+using the F105W and F125W filters as well as the UVIS detector using the F475W.
+All these observations fall within the SkyCell at position **x07y19** in the
+ProjectionCell **p1889**, but given the dramatic plate scale differences, these observations can not be used to create a
+single mosaic.
+
+.. code:: python
+
+    with open('skycell_p1889x07y19_input.txt', 'w') as fout:
+        _ = [fout.write(f'{file}\n') for file in p1889_scells['skycell-p1889x07y19'].members]
+
```

Comment:
I am not sure why this partial version of the example is here again.    Also, p1889_scells should be sky_cells_dict.

Reply:
Fixed in 3cd7f7e.